### PR TITLE
Add throw expressions

### PR DESCRIPTION
**Pull request by jridgewell**
Stage 2 proposal: https://github.com/tc39/proposal-throw-expressions

### CODE REVIEW

**Comment by nicolo-ribaudo** [comment]
Can we make `babel-generator` print it in one line, like we do for helpers? I think it looks a little nicier.
```js
const a = throw new Error();

const a = (function(e) { throw e; })(new Error());
// versus
const a = (function (e) {
  throw e;
})(new Error());
```

**Comment by jridgewell** [comment]
> Can we make babel-generator print it in one line, like we do for helpers

I thought I removed that code.

**Comment by hzoo** [comment]
Can add to stage 2 preset now/later?

File: packages/babel-plugin-transform-throw-expressions/src/index.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import syntaxThrowExpressions from "babel-plugin-syntax-throw-expressions";
+
+export default function({ types: t }) {
+  return {
+    inherits: syntaxThrowExpressions,
+
+    visitor: {
+      UnaryExpression(path) {
+        const { operator, argument } = path.node;
+        if (!operator === "throw") return;
```
Comment (nicolo-ribaudo) [inline packages/babel-plugin-transform-throw-expressions/src/index.js]: Needs parens or !==

Comment (jridgewell) [reply packages/babel-plugin-transform-throw-expressions/src/index.js]: lololololololololololololololololololol

**Pull request merged**